thirdeye-server/src/main/java/ai/startree/thirdeye/auth/AuthorizationManager.java

```diff
@@ -58,6 +62,12 @@ public class AuthorizationManager {
   private final EnumerationItemManager enumerationItemManager;
   private final AnomalyManager anomalyManager;
 
+  // cache resource id's to their respective auth namespaces
+  private final Cache<Long, String> namespaceCache = CacheBuilder.newBuilder()
+      .maximumSize(2048)
+      .expireAfterAccess(5, TimeUnit.SECONDS)
```

Comment:
How often are people changing policies?

Reply:
🤔 I'm not sure. I think I'd rather have the update/create methods add to the cache instead of expiring like this.